gammapy/modeling/models/time.py

```diff
@@ -123,6 +125,55 @@ def evaluate_norm_at_phase(self, phase):
         fp = self.table["NORM"]
         return np.interp(x=phase, xp=xp, fp=fp, period=1)
 
+    def sample_time(self, n_events, t_min, t_max, t_delta="1 s", random_state=0):
+        """Sample arrival times of events.
+
+        Parameters
+        ----------
+        n_events : int
+            Number of events to sample.
+        t_min : `~astropy.time.Time`
+            Start time of the sampling.
+        t_max : `~astropy.time.Time`
+            Stop time of the sampling.
+        t_delta : `~astropy.units.Quantity`
+            Time step used for sampling of the temporal model.
+        random_state : {int, 'random-seed', 'global-rng', `~numpy.random.RandomState`}
+            Defines random number generator initialisation.
+            Passed to `~gammapy.utils.random.get_random_state`.
+
+        Returns
+        -------
+        time : `~astropy.units.Quantity`
+            Array with times of the sampled events.
+        """
+        time_unit = u.second
+
+        t_min = Time(t_min)
+        t_max = Time(t_max)
+        n_events = n_events
```

Comment:
This statement is without effect, just remove...

Reply:
Done.